webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -86,19 +75,21 @@ export const getFiles = params => get({
   errorToast: error => __(`Something went wrong while fetching files! ${getResponseErrorMsgs(error.response)}`),
 });
 
-export const updateContentView = (cvId, params) => dispatch => dispatch(put({
+export const updateContentView = (cvId, params, handleSuccess) => put({
   type: API_OPERATIONS.PUT,
   key: cvDetailsKey(cvId),
   url: api.getApiUrl(`/content_views/${cvId}`),
-  params,
-  handleSuccess: response => cvUpdateSuccess(response, dispatch),
-  handleError: error => dispatch(apiError(null, error)),
+  handleSuccess,
+  params: { include_permissions: true, ...params },
+  successToast: () => __(' Content view updated'),
+  errorToast: error => error,
+  updateData: (prevState, respState) => ({ ...prevState, ...respState }),
   actionTypes: {
```

Comment:
This works too:
`updateData: (prevState, respState) => respState,`

Reply:
Sweet